sys-cluster/kube-router/files/kube-router.initd

```diff
@@ -20,3 +20,13 @@ extra_commands="clear"
 clear() {
     "${command}" --cleanup-config
 }
+
+if [[ -n "${KUBE_ROUTER_HEALTHCHECK_URI}" ]]; then
+        healthcheck_delay=60
+        healthcheck_timer=30
+
+        healthcheck() {
+                command -v wget || return 0
+                wget -Oq- "${KUBE_ROUTER_HEALTHCHECK_URI}"
+        }
+fi
```

Comment:
Will this be backwards-compatible?

Reply:
Yes, new confd and init.d can and should be used with previous version also